### PR TITLE
Case studies

### PR DESCRIPTION
**Pull request by nerucheva**
This pull request adds the case studies page.

**To do when the updates are got**
- [x] adjust the case studies card for mobiles (320px)
- [x] add a link to this page to the footer
- [ ] add SEO data

[Preview](https://deploy-preview-267--ebpfio.netlify.app/case-studies/)

### CODE REVIEW

File: src/pages/case-studies.js [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+    links: [
+      {
+        name: "Blog",
+        href: "https://inf.news/en/tech/3313635d67f796ae38e7ef64061bc7c4.html",
+      },
+    ],
+  },
+  {
+    logo: F5,
+    description: ["F5", " BIG-IP integration with Cilium VTEP feature"],
```
Comment (vannyle) [inline src/pages/case-studies.js:322]: It seems easier to insert `html` to the description than to use an array. 
```suggestion
    description: "<strong>F5</strong> BIG-IP integration with Cilium VTEP feature",
```

File: src/pages/case-studies.js [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+    <div className="card-logo">
+      <img src={logo} />
+    </div>
+    <div className="card-info">
+      <div className="card-description-wrapper">
+        <p className="card-description">
+          {description.map((item, i) =>
+            i === 0 ? <strong>{item}</strong> : <>{item}</>
+          )}
+        </p>
```
Comment (vannyle) [inline src/pages/case-studies.js:347]:
```suggestion
        <p className="card-description"  dangerouslySetInnerHTML={{__html: description}} />
```

**Comment by xmulligan** [comment]
@nerucheva this is coming along well! We would like to edit the design. Can you remove the text from the boxes and make the logos larger. Also instead of the words, video 1, can we instead have an icon representing the thing it is (paper for blog, video camera for video, ect)?

also besides adding it to the footer, can you also add it to the menu under the community menu?

**Review by vannyle — APPROVED** [review]
Great job! @nerucheva

**Comment by nerucheva** [comment]
> 
Hi! Our designers tried other variants, but they don't work. So we recommend using the first version of the design.

**Comment by borkmann** [comment]
@nerucheva Thanks, this looks nice!

Just few small nits:
- in the "Community" menu the "Case Studies" should come after "eCHO" entry
- in the footer under "Get Started", the "Case Studies" should come after "What is eBPF"
- In the "[...] please submit a pull request. [...]" we should link "pull request" to https://github.com/ebpf-io/ebpf.io

I was also thinking that it would be nice to have something better than "Video 1", "Video 2", "Blog 1", "Blog 2", etc.

I did a small draft, perhaps something like the below could work where we list external references e.g. via symbol:

edit

But also open to other ideas.

Great work, thanks!

**Comment by nerucheva** [comment]
> I was also thinking that it would be nice to have something better than "Video 1", "Video 2", "Blog 1", "Blog 2", etc.

@borkmann Hi! I made changes to the menus and added the link. What about the design changes, won't you mind leaving feedback under [this task in notion](https://www.notion.so/isovalent/eBPF-Case-Study-page-b452def50d494155aab34ec1488f7c71)?

**Comment by borkmann** [comment]
> > I was also thinking that it would be nice to have something better than "Video 1", "Video 2", "Blog 1", "Blog 2", etc.
> 
> @borkmann Hi! I made changes to the menus and added the link. What about the design changes, won't you mind leaving feedback under [this task in notion](https://www.notion.so/isovalent/eBPF-Case-Study-page-b452def50d494155aab34ec1488f7c71)?

Ok, done. Meanwhile we can move forward and get this page merged. Thanks!

**Review by borkmann — APPROVED** [review]

**Review by xmulligan — APPROVED** [review]

**Pull request merged**